eth/handler.go

```diff
@@ -321,21 +321,9 @@ func newHandler(config *handlerConfig) (*handler, error) {
 
 	broadcastBlockWithCheck := func(block *types.Block, propagate bool) {
 		if propagate {
-			checkErrs := make(chan error, 2)
-
-			go func() {
-				checkErrs <- core.ValidateListsInBody(block)
-			}()
-			go func() {
-				checkErrs <- core.IsDataAvailable(h.chain, block)
-			}()
-
-			for i := 0; i < cap(checkErrs); i++ {
-				err := <-checkErrs
-				if err != nil {
-					log.Error("Propagating invalid block", "number", block.Number(), "hash", block.Hash(), "err", err)
-					return
-				}
+			if err := core.IsDataAvailable(h.chain, block); err != nil {
+				log.Error("Propagating block with invalid sidecars", "number", block.Number(), "hash", block.Hash(), "err", err)
+				return
```

Comment:
it could change the behavior of block check in https://github.com/bnb-chain/bsc/blob/master/eth/fetcher/block_fetcher.go#L929
block will be broadcasted without check these list.

Reply:
> it could change the behavior of block check in https://github.com/bnb-chain/bsc/blob/master/eth/fetcher/block_fetcher.go#L929 block will be broadcasted without check these list.

but indeed, they are useless, because
in func handleBlockBodies and DeliverBodies, these 3 lists have been checked.
in func handleNewBlock, Uncles, Transactions have been checked,
so only add check for Withdrawalsin func handleNewBlock` is ok.